dask/dataframe/tests/test_groupby.py

```diff
@@ -3632,3 +3632,24 @@ def test_groupby_numeric_only_true(func):
         ddf_result = getattr(ddf.groupby("A"), func)(numeric_only=True)
         pdf_result = getattr(df.groupby("A"), func)(numeric_only=True)
         assert_eq(ddf_result, pdf_result)
+
+
+@pytest.mark.skipif(not PANDAS_GT_150, reason="numeric_only not supported for <1.5")
+@pytest.mark.parametrize("func", ["cov", "corr"])
+def test_groupby_numeric_only_false_cov_corr(func):
+    df = pd.DataFrame(
+        {
+            "float": [1.0, 2.0, 3.0, 4.0, 5, 6.0, 7.0, 8.0],
+            "int": [1, 2, 3, 4, 5, 6, 7, 8],
+            "timedelta": pd.to_timedelta([1, 2, 3, 4, 5, 6, 7, 8]),
```

Comment:
I see you have special handling for `datetime` as well, maybe include it in your test data, to ensure it works.

Reply:
dt is buggy in pandas unfortunately, so we get back garbage. I am planning on opening an issue there, but testing doesn't make much sense till then

Reply:
👍